packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -1055,7 +1055,7 @@ export class PersonalDetailsForm extends React.Component {
                                                 id='email_consent'
                                                 defaultChecked={!!values.email_consent}
                                                 disabled={!this.isChangeableField('email_consent') && !is_virtual}
-                                                className={is_dashboard && 'dc-checkbox-blue'}
+                                                className={is_dashboard ? 'dc-checkbox-blue' : ''}
```

Comment:
Since we are using `classNames` library in deriv, I think it is cleaner to get help from this library anywhere possible:

```js
import classNames from 'classnames'; 

className={classNames({
    'dc-checkbox-blue': is_dashboard,
})}
```